empire/empire.go

```diff
@@ -416,19 +409,23 @@ func newCertManager(config *aws.Config) sslcert.Manager {
 	return sslcert.NewIAMManager(config, "/empire/certs/")
 }
 
-func newRunner(options RunnerOptions, s *store) *runner {
-	var r containerRelayer
-	if options.API == "fake" {
-		log.Println("warn: runner not configured, command runner disabled.")
-		r = &fakeRelayer{}
-	} else {
-		r = &relayer{API: options.API}
+func newRunner(o DockerOptions, s *store) (*runnerService, error) {
```

Comment:
We're initializing a lot of docker clients now. It could be better to initialize one and pass it down here.